definitions.md

```diff
@@ -0,0 +1,50 @@
+# Definition of Common Terms
+
+## Infrastructure
+
+- <a name="hub"/>**Hub**:
+    - **Definition**: A kubernetes cluster that is running OCM, typically connected to one or more [spokes](#spoke).
```

Comment:
What about linking out to https://multicluster.sigs.k8s.io/concepts/work-api/, which defines some of these terms, and give OCM as an example implementation that kuadrant uses.